### PR TITLE
Timezones feature

### PR DESCRIPTION
**Pull request by huebim**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/PhotoboothProject/photobooth/blob/dev/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "x" next to an item)

- [ ] Documentation update
- [ ] Bug fix
- [x] New feature
- [ ] Other, please explain:

I think it will work, but I couldn't try it out on a real system to see whether the file names actually get the locale time as a name.

#### What changes did you make? (Give an overview)
timezons for php in a data file timezones.php
add admin panel option in general
update index.php to set the choice at startup

### CODE REVIEW

File: src/Configuration/PhotoboothConfiguration.php [hunk 1]
```diff
@@ -997,13 +998,20 @@ protected function addAdminPanel(): NodeDefinition
 
     protected function addUiNode(): NodeDefinition
     {
+        include PathUtility::getAbsolutePath('/src/timezones.php');
```
Comment (andi34) [inline src/Configuration/PhotoboothConfiguration.php:1001]: include PathUtility::getAbsolutePath('src/timezones.php');

File: index.php [hunk 2]
```diff
@@ -9,6 +9,10 @@
 
 $assetService = AssetService::getInstance();
 
+if ($config['ui']['local_timezone']) {
```
Comment (andi34) [inline index.php]: Enough to put inside lib/boot.php ? Else we also need it on chroma etc

Comment (huebim) [reply index.php]: ok, is a better way.

File: src/Configuration/PhotoboothConfiguration.php (hunk 1 above)
Comment (huebim) [reply src/Configuration/PhotoboothConfiguration.php:1001]: this timezone array is in this protected function not available

File: index.php (hunk 2 above)
Comment (huebim) [reply index.php]: i have the problem, when in php.ini is set date.timezone then its override the date_default_timezone_set. if I'm not mistaken

**Comment by andi34** [comment]
https://github.com/PhotoboothProject/photobooth/pull/985

**Pull request closed without merging**